src/Test/Diagnostics/Roslyn.Hosting.Diagnostics.csproj

```diff
@@ -25,6 +25,7 @@
     <Reference Include="System.Xaml" />
     <Reference Include="System.Xml" />
     <Reference Include="WindowsBase" />
+    <PackageReference Include="Microsoft.VisualStudio.Utilities" Version="$(MicrosoftVisualStudioUtilitiesVersion)" />
```

Comment:
Why is this one needed? There's no other PackageReferences in the entire project?

Reply:
It seems it is not necessary. Removing. Thank you!